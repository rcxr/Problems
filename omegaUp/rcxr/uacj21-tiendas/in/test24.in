10
2
6
6 entrar centro-comercial
3 entrar la-mera-mera-michoacana
3 entrar ajedreces-pa-todos
5 entrar centro-comercial
2 entrar la-autentica-michoacana
3 entrar la-gran-michoacana
