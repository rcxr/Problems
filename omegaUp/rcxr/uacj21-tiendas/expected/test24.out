ajedreces-pa-todos centro-comercial la-gran-michoacana la-mera-mera-michoacana
